.github/workflows/bench-pr.yml

```diff
@@ -129,6 +129,9 @@ jobs:
 
       - name: Run ${{ matrix.benchmark.name }} benchmark
         shell: bash
+        env:
+          RUST_BACKTRACE: full
+          LD_LIBRARY_PATH: target/duckdb-v1.3.0
```

Comment:
Setting this env var is not required locally. It's not clear to me yet why this is necessay on CI. The current setup otherwise worked locally on macOS and Linux (Docker container).